resources/views/api/includes/sections/top_nav_user.blade.php

```diff
@@ -1,3 +1,4 @@
+<div class="row">
 <div class="navbar-fixed">
     <nav role="navigation">
         <div class="nav-wrapper">
```

Comment:
Where is this particular file being used in this project ?.. `top_nav_user.blade.php`